api_tests/actions/views/test_action_list.py

```diff
@@ -319,8 +325,11 @@ def test_valid_transitions(
                 preprint.date_last_transitioned = None
                 preprint.save()
                 payload = self.create_payload(preprint._id, trigger=trigger)
-                with capture_notifications():
```

Comment:
This has to covered in mocked and non-mocked state due to the parameterization of tests